_includes/head.html

```diff
@@ -17,5 +17,5 @@
 
 <!-- For all browsers -->
 <link rel="stylesheet" href="{{ base_path }}/assets/css/main.css">
-
-<meta http-equiv="cleartype" content="on">
\ No newline at end of file
+<link rel="stylesheet" href="{{ base_path }}/assets/css/primer.css"
```

Comment:
@hectorsector, I think this should be

```html
<link rel="stylesheet" href="{{ base_path }}/assets/css/primer.css"
<link rel="stylesheet" href="{{ base_path }}/assets/css/main.css"> 
```

That way it will load primer and **then** load our customisations from `main.css`.